tests/unit_test_ortho_functions.py

```diff
@@ -532,11 +554,12 @@ def __init__(self, fn, vendor=None, sat=None):
         TestWriteMetadata,
         TestCollectFiles,
         TestDEMOverlap,
+        TestAutoDEMOverlap,
         TestTargetExtent,
         TestOutputDataValues,
         TestRPCHeight,
         TestCalcEarthSunDist,
-        TestOverlapCheck
+        TestOverlapCheck,
```

Comment:
I think we don't want a comma for the last item in this list

Reply:
Should not matter.